rsts/howto/sandbox.rst

```diff
@@ -21,8 +21,95 @@ Flyte can be run using a Kubernetes cluster only. This installs all the dependen
 Deploy Flyte Sandbox environment locally - on your laptop
 *********************************************************
 
-Flyte Sandbox deployment can be run locally on your laptop. This will allow only some of the functionality, but is great to get started with.
-Refer to :ref:`tutorials-getting-started-run-on-flyte`.
+Ensure ``kubectl`` is installed. Follow `kubectl installation docs <https://kubernetes.io/docs/tasks/tools/install-kubectl/>`_. On Mac::
+
+    brew install kubectl
+
+
+
+.. tabs::
+
+    .. tab:: Use flyte-sandbox docker image (Recommended)
```

Comment:
Maybe just these as titles to make them fit in "regular" screen sizes?
"Docker, K3d, Docker Desktop, Minikube, EKS/GKE"

Reply:
Given there is a separate section below for EKS/GKE, maybe we don't need the tab here at all?